### PR TITLE
Configuration id fix

### PR DESCRIPTION
**Pull request by remmintan**

### CODE REVIEW

File: Core/Tenders.Core/Services/ModelBaseFactory.cs [hunk 1]
```diff
@@ -34,5 +36,25 @@ public T MakeInstance<T>()
             result.CreatedDate = this.dateTimeService.MoscowTime();
             return result;
         }
+
+        /// <inheritdoc/>
+        public TTo MakeInstnaceFrom<TFrom, TTo>(TFrom datamodel)
```
Comment (DamienVStone) [inline Core/Tenders.Core/Services/ModelBaseFactory.cs]: Instnace => Instance

File: Core/Tenders.Core/Services/ModelMapperService.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            this.factory = factory;
+        }
+
+        /// <inheritdoc/>
+        public TTo Map<TFrom, TTo>(TFrom sourceModel)
+            where TTo : TFrom
+        {
+            var result = this.factory.MakeInstance<TTo>();
+
+            var properties = typeof(TFrom).GetProperties();
```
Comment (DamienVStone) [inline Core/Tenders.Core/Services/ModelMapperService.cs]: Split using SRP

File: tests/Tenders.Core.Tests/ModelBaseFactoryTest.cs [hunk 3]
```diff
@@ -47,6 +54,30 @@ public void CreateInstanse_ModelBaseFactory_ReturnedObject()
...
+            var mbf = new ModelBaseFactory(factoryService, dateTimeService, mapperService);
+
+            // act
+            var result = mbf.MakeInstnaceFrom<INote, Note>(mockNote);
+
+            // assert
+            Assert.Equal(mockNote.Text, result.Text);
+            Assert.True(result.IsActive);
+            Assert.True(Guid.TryParse(result.Id, out var res));
+            Assert.True((DateTime.Now - result.CreatedDate).TotalMinutes < 1);
```
Comment (DamienVStone) [inline tests/Tenders.Core.Tests/ModelBaseFactoryTest.cs]: What?!?!

File: tests/Tenders.Core.Tests/ModelMapperServiceTest.cs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        /// Tests successful map from interface to model.
+        /// </summary>
+        [Fact]
+        public void MapTest()
+        {
+            // arrange
+            var factoryService = new FactoryBase();
+            var mapperService = new ModelMapperService(factoryService);
+            INote sourceModel = new Note();
+            sourceModel.Text = "Тестовое сообщение";
```
Comment (DamienVStone) [inline tests/Tenders.Core.Tests/ModelMapperServiceTest.cs]: English please

File: Core/Tenders.Core/Services/ModelMapperService.cs (hunk 2 above)
Comment (remmintan) [reply Core/Tenders.Core/Services/ModelMapperService.cs]: Should I extract Instance creation from ModelMapperService?

**Pull request merged**